apps/test/unit/templates/CertificateTest.js

```diff
@@ -78,7 +78,7 @@ describe('Certificate', () => {
       server.restore();
     });
 
-    it('renders using pegasus without experiment', () => {
+    it('renders certificate pages using code studio urls', () => {
```

Comment:
this diff looks weird. what I did was remove "renders using pegasus without experiment", and then rename "renders using code studio with experiment" to this name.